### DISCUSSION TITLE
Colors and styles of generated html content like S[bhqs]

### DISCUSSION THREAD

**Discussion by aquaherd** [qutebrowser/qutebrowser#6330] [Q&A]
Is it possible to style this content like bookmart list, history and settings pages without resorting to dark modes?

**Comment by rien333** [comment]
@The-Compiler perhaps qutebrowser can add a unique css class to those pages (like the [readability scripts currently do](https://github.com/qutebrowser/qutebrowser/commit/55fdae8eed0f93152b5676cca77b6243fd95d9b0)), such that user can make custom changes through a stylesheet specified in `content.user_stylesheets`?

**Reply by The-Compiler** [reply]
I guess that'd work, but it depends a bit on what @aquaherd's goal is precisely.

**Comment by The-Compiler** [comment]
Can you elaborate a bit on what your use-case is? How/why do you want to style them?

**Reply by aquaherd** [reply]
Most sites I visit either accept my preference for dark themes or are configured to use a dark theme of their own so that I can get by without using QBs dark mode overrides. However when I open the qutebrowser generated pages the white canvas is a real pain in the eyes. I would like to set background, text and link colors just like in the menu.

Also, since I am using qutebrowser on a github-proliferated settings.yml on Linux, Mac and Windows, I'd like to have a consistent appearance in fonts. I noted that one platform would pick a sans-serif font I like while the other does not.
I know I can set a default for sans-serif globally to catch this, but this would apply to the entire web not the generated content.

If possible, it would be imho best if these generated pages somehow had an overall theme-ability like the status bar, tabs bar and so forth. Otherwise I would be happy if the generated HTML content would e.g. include some qute://bookmarks/style.css stylesheet instead of the current verbatim in-file generation and we could somehow edit this file to our liking.
